packages/block-library/src/navigation/view.js

```diff
@@ -1,35 +1,33 @@
 /**
  * WordPress dependencies
  */
-import { store } from '@wordpress/interactivity';
+import { store as wpStore } from '@wordpress/interactivity';
 
 const focusableSelectors = [
 	'a[href]',
-	'area[href]',
 	'input:not([disabled]):not([type="hidden"]):not([aria-hidden])',
 	'select:not([disabled]):not([aria-hidden])',
 	'textarea:not([disabled]):not([aria-hidden])',
 	'button:not([disabled]):not([aria-hidden])',
-	'iframe',
-	'object',
-	'embed',
 	'[contenteditable]',
 	'[tabindex]:not([tabindex^="-"])',
 ];
 
-const openMenu = ( { context, ref }, menuOpenedOn ) => {
-	context.core.navigation.isMenuOpen[ menuOpenedOn ] = true;
+const openMenu = ( store, menuOpenedOn ) => {
+	const { context, ref, selectors } = store;
+	selectors.core.navigation.menuOpenBy( store )[ menuOpenedOn ] = true;
 	context.core.navigation.previousFocus = ref;
-	if ( context.core.navigation.overlay ) {
+	if ( context.core.navigation.type === 'overlay' ) {
 		// Add a `has-modal-open` class to the <html> root.
 		document.documentElement.classList.add( 'has-modal-open' );
 	}
 };
 
-const closeMenu = ( { context, selectors }, menuClosedOn ) => {
-	context.core.navigation.isMenuOpen[ menuClosedOn ] = false;
+const closeMenu = ( store, menuClosedOn ) => {
+	const { context, selectors } = store;
+	selectors.core.navigation.menuOpenBy( store )[ menuClosedOn ] = false;
 	// Check if the menu is still open or not.
-	if ( ! selectors.core.navigation.isMenuOpen( { context } ) ) {
+	if ( ! selectors.core.navigation.isMenuOpen( store ) ) {
```

Comment:
I started passing `store` instead of `{context}` to the selector. I think it's safer to do it that way by default.